### DISCUSSION TITLE
That empty lines are not printed and that the space is adjusted

### DISCUSSION THREAD

**Discussion by jtejedaDev** [lukevp/ESC-POS-.NET#215] [General]
How can I Send cancellation command
So that it doesn't print that line, I have a ternary conditional but if both values ​​inside the printline are false I don't want it to print but I want that space to be removed and automatically adjusted.

thank you

**Comment by igorocampos** [comment]
There's no cancellation command. What you want to do instead is not send printline at all if you don't want anything printed...
